plugin/src/main/java/org/opensearch/ml/model/MLModelCacheHelper.java

```diff
@@ -386,9 +386,11 @@ public String[] getLocalDeployedModels() {
      */
     public String[] getExpiredModels() {
```

Comment:
Do we have any test for this method?

Reply:
The syncUpCron will cover this part. Only an unit tests will not find this edge case. It has to be in the scenario of "local model deployed and wait more than 10 seconds to predict".